jest-transformer.js

```diff
@@ -1,2 +1,5 @@
 const babelPreset = require(`babel-preset-gatsby-package`)()
-module.exports = require(`babel-jest`).createTransformer(babelPreset)
+module.exports = require(`babel-jest`).createTransformer({
```

Comment:
I would add a test environment to babel-reset-gatsby-package to include it so other plugins have access to it too.

https://babeljs.io/docs/en/options#env

Reply:
Added it to `babel-preset-gatsby` so that other folks have access to it for their unit tests. Wanted to avoid adding it to `babel-preset-gatsby-package` for now after a conversation with @pieh because it doesn't seem necessary since this `jest-transformer.js` gets run for all tests at the moment.

Reply:
We could ship it like this and re-iterate